encrypted-media-respec.html

```diff
@@ -2816,7 +2816,7 @@
         <section>
           <h4><a>MediaEncryptedEventInit</a></h4>
           <div><pre class="idl">dictionary MediaEncryptedEventInit : EventInit {
-             DOMString    initDataType = "";
+             required DOMString    initDataType
```

Comment:
`initData` was already nullable on interface and init dict, so this seemed fine before, I think it was only because I assumed that the UA would never fire events with a null `initData` attribute that it seemed like a change to `MediaEncryptedEventInit` might be needed.